### PR TITLE
install and setup tailwindcss

### PR DESCRIPTION
**Pull request by iamCelott**
image
image

### CODE REVIEW

File: src/frontend/index.css
Comment (pahrizal) [inline src/frontend/index.css:1]: remove all unused css style. only the `@tailwind...` in this file

File: package-lock.json
Comment (pahrizal) [inline package-lock.json:1]: remove this file. use `yarn` for package manager. when installing package, avoid using `npm install`, use `yarn add <package_name>`

**Review by pahrizal — CHANGES_REQUESTED** [review]
please made changes accordingly.

File: package-lock.json
Comment (iamCelott) [reply package-lock.json:1]: sorry sir, did you mean to delete package-lock.json file??

**Comment by pahrizal** [comment]
Yes, you’re right. Delete package-lock.json. This file generated when you use `npm i …` commandSent from my iPhoneOn 24 Jan 2024, at 12:39 am, iamCelott ***@***.***> wrote:﻿
@iamCelott commented on this pull request.



On package-lock.json:
sorry sir, did you mean to delete package-lock.json file??

**Review by pahrizal — APPROVED** [review]
Look awesome! You need a review from your one of your team mates to merge this 🎉

**Comment by pahrizal** [comment]
Please ask @juhan26 for a review

**Review by Azissa — APPROVED** [review]
awesome can

**Comment by juhan26** [comment]
> Please ask @juhan26 for a review

already sir, good job @iamCelott 👍🏼

**Review by juhan26 — APPROVED** [review]
good job @iamCelott

**Pull request merged**